### ISSUE TITLE
SVG Utils class's temp files violate future Batik Transcoder security guidelines

### ISSUE THREAD

**Issue by bobwoco** [magefree/mage#11241]
While testing the XML Batik transcoder upgrade for #10968 , I ran into an issue where when attempting to add additional CSS to svg icons depending on the theme, the following exception occured:
```
java.lang.SecurityException: Could not access the current document URL when trying to load an external resource file:/D:/Documents/mage/Mage.Client/plugins/images/symbols/temp/mana-svg-settings.css. The external resource will not be loaded as it is not possible to verify it comes from the same location as the document.
	at org.apache.batik.bridge.DefaultExternalResourceSecurity.checkLoadExternalResource(DefaultExternalResourceSecurity.java:63)
	at org.apache.batik.bridge.UserAgentAdapter.checkLoadExternalResource(UserAgentAdapter.java:391)
	at org.apache.batik.bridge.BridgeContext.checkLoadExternalResource(BridgeContext.java:1890)
	at org.apache.batik.css.engine.CSSEngine.parseStyleSheet(CSSEngine.java:1166)
	at org.apache.batik.css.engine.CSSEngine.parseStyleSheet(CSSEngine.java:1116)
	at org.apache.batik.bridge.BridgeContext.initializeDocument(BridgeContext.java:390)
	at org.apache.batik.bridge.GVTBuilder.build(GVTBuilder.java:55)
	at org.apache.batik.transcoder.SVGAbstractTranscoder.transcode(SVGAbstractTranscoder.java:210)
	at org.apache.batik.transcoder.image.ImageTranscoder.transcode(ImageTranscoder.java:92)
	at org.apache.batik.transcoder.XMLAbstractTranscoder.transcode(XMLAbstractTranscoder.java:142)
	at org.apache.batik.transcoder.SVGAbstractTranscoder.transcode(SVGAbstractTranscoder.java:158)
	at org.mage.card.arcane.SvgUtils.loadSVG(SvgUtils.java:158)
	at org.mage.card.arcane.ManaSymbols.loadSymbolAsSVG(ManaSymbols.java:236)
	at org.mage.card.arcane.ManaSymbols.lambda$loadSymbolImages$0(ManaSymbols.java:307)
	at java.util.stream.ForEachOps$ForEachOp$OfInt.accept(ForEachOps.java:205)
	at java.util.stream.Streams$RangeIntSpliterator.forEachRemaining(Streams.java:110)
	at java.util.Spliterator$OfInt.forEachRemaining(Spliterator.java:693)
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:482)
	at java.util.stream.ForEachOps$ForEachTask.compute(ForEachOps.java:291)
	at java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:731)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1067)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1703)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:172)
```
After doing some research, it looks like after we do that upgrade, we'll have to either loosen the security mode that we use to allow for external files or refactor this code to not pull the CSS settings in from external files. (see [here](https://issues.apache.org/jira/browse/BATIK-1333) for the commit that changed the behaviour and for explanation of the change see [here](https://xmlgraphics.apache.org/batik/using/scripting/security.html)) @JayDi85 is there any reason why it wouldn't be better to just get the CSS at runtime? We already have `getCardIconsCssSettings` to define the specific settings that we would need, would there be any downside to getting the CSS from that rather than writing it to disk and then reading it again? It seems likely that loosening the security mode wouldn't be best practice.

**Comment by JayDi85** [comment]
I used temp css file cause svg settings (needs for nice svg rendering) works with file only. If you can fix it without temporary file then it can be good solution (e.g. setup svg settings by commands or memory only resources/streams).

**Comment by JayDi85** [comment]
Here:

![IMG_8946](https://github.com/magefree/mage/assets/8344157/e401c155-13b3-4317-b285-3825add8e9c3)

**Comment by JayDi85** [comment]
One of the solution: change files structure for svg icons. Example: store it as current version do, but prepare and use real files from a temp folder with css styles). I will look at it later.

**Comment by xenohedron** [comment]
addressed in #11366
